src/library/Hooks/useInflation/index.tsx

```diff
@@ -35,20 +38,24 @@ export const useInflation = () => {
       stakeTarget -
       Math.min(auctionMax, numAuctions.toNumber()) * auctionAdjust;
     const idealInterest = maxInflation / idealStake;
-    const inflation =
-      100 *
-      (minInflation +
-        (stakedFraction <= idealStake
-          ? stakedFraction * (idealInterest - minInflation / idealStake)
-          : (idealInterest * idealStake - minInflation) *
-            2 ** ((idealStake - stakedFraction) / falloff)));
+
+    const totalIssuanceInTokens = totalIssuance
+      .div(BN_BILLION)
+      .div(BN_THOUSAND);
+
+    const inflation = totalIssuanceInTokens.isZero()
+      ? 0
+      : 100 * (yearlyInflationInTokens / totalIssuanceInTokens.toNumber());
+
+    let stakedReturn = stakedFraction ? inflation / stakedFraction : 0;
```

Comment:
Do you know why on azero.dev we have slightly different value?

![image](https://user-images.githubusercontent.com/3909333/204994892-88966603-cbfd-46a9-9a7d-645da3c60093.png)



![image](https://user-images.githubusercontent.com/3909333/204994806-39ea3a3c-deb1-47ba-9aea-9abe4d61464e.png)

Reply:
Polkadot js uses the sum over `api.derive.staking.electedInfo.info[filter on isActive].totalBond`
The dashboard uses `[api.query.staking.erasTotalStake, previousEra]`
I guess the difference may come from this?